pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/main/java/org/apache/pinot/plugin/stream/kinesis/KinesisConsumer.java

```diff
@@ -159,23 +159,23 @@ private KinesisRecordsBatch getResult(StreamPartitionMsgOffset startOffset, Stre
 
       return new KinesisRecordsBatch(recordList, startShardToSequenceNum.getKey(), isEndOfShard);
     } catch (IllegalStateException e) {
-      LOGGER.warn("Illegal state exception, connection is broken", e);
+      LOGGER.warn("Illegal state exception, connection is broken: {}", e.getMessage());
```

Comment:
would it just be simpler to make these logs at `debug` level? Are these exception messages useful without their stacktrace?

Reply:
makes sense. The only ones not `debug` level now are which indicate a major error like Kinesis stream is down or AWS credentials wrong, etc.